docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+
+Nginx allows you to add per-site configurations to individual files in a subdirectory called `sites-available/`. Using your favorite text editor, create a new Nginx configuration at `/etc/nginx/sites-available/temporal-ui`:
+
+```bash
+sudo vim /etc/nginx/sites-available/temporal-ui
+```
+
+Paste the following into the new configuration file, being sure to replace `your_domain` with your domain name.
+
+```
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    server_name your_domain www.your_domain
+
+    access_log /var/log/nginx/temporal.access.log;
+    error_log /var/log/nginx/temporal.error.log;
+
+    location / {
+        proxy_pass http://127.0.0.1:8233;
+        proxy_http_version 1.1;
+        proxy_read_timeout 300;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Real-PORT $remote_port;
+        allow your_ip_addresses;
+        deny all;
+    }
+
+    listen 443 ssl;
+    # RSA certificate
+    ssl_certificate /etc/letsencrypt/live/your_domain/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/your_domain/privkey.pem;
+
+    # Redirect non-https traffic to https
+    if ($scheme != "https") {
+        return 301 https://$host$request_uri;
+    }
+}
+```
+
+You can read this configuration as having three main “blocks” to it. The first block, coming before the `location /` line, contains a boilerplate Nginx configuration for serving a website on the default HTTP port, 80. The `location /` block contains a configuration for proxying incoming connections to the Temporal Web UI, running on port 8233 internally, while preserving SSL. The configuration at the end of the file, after the `location /` block, loads your LetsEncrypt SSL keypairs and redirects HTTP connections to HTTPS.
+
+Note the `allow your_ip_address;` line. You should replace this with the IP address that you'll need to access the Temporal Web UI from. You can add additional addresses in the same range using [CIDR notation](https://www.digitalocean.com/community/tutorials/understanding-ip-addresses-subnets-and-cidr-notation-for-networking), or add additional consecutive `allow` lines for multiple IPs. The `deny all;` on the following line will block traffic to all but the specified IPs. Because Temporal does not use any kind of authentication by default, restricting traffic by IP address is the least complex way of providing secure access.
```

Comment:
Yeah, I thought about adding a whole firewall step, but I'm reluctant to do so because we'd have to be somewhat opinionated about the choice of firewall, and it's technically an unnecessary dependency when Nginx can do this.